### ISSUE TITLE
Edit compose files live

### ISSUE THREAD

**Issue by xcad2k** [nuxxapp/nuxx#22]
It's nice to be able to import am existing compose file, but it's not a good UX to upload a file somewhere and provide an import URL.

A much better approach would be to just be able to live to edit the compose files to simplify importing via copy/paste.

**Comment by corpulent** [comment]
@xcad2k yea this is part two of importing,  to be able to edit yaml code.  Will try to push it out this week.

**Comment by xcad2k** [comment]
Great news 👍

**Comment by basudip** [comment]
> 
> 
> @xcad2k yea this is part two of importing, to be able to edit yaml code. Will try to push it out this week.

great, looking forward to it.

**Comment by corpulent** [comment]
@basudip @xcad2k This feature is deployed,  please give it a try and test things out.  Please add all bugs you see here.

**Comment by basudip** [comment]
Hi, Tested it working expected. The tool is now absolutely necessary for my docker builds. 

**Some Issues I found**
1. Network not imported properly while changing compose version to 1 from version 2/3. No issue in the GUI.
2. 'devices' not importing in any version of compose. maybe it's not implemented yet?
3. version dropdown should auto change after pasting and hit done button. Currently stays the same. 

These are all my quick find. will update the list if I found anything new.
